recipe/meta.yaml

```diff
@@ -89,7 +91,11 @@ outputs:
         - libcxx >={{ cxx_compiler_version }}.a0  # [osx]
     test:
       commands:
+        {% if git_commit != "" %}
+        - test -f $PREFIX/lib/libLLVM-{{ major_ver }}git${SHLIB_EXT}  # [unix]
```

Comment:
So you want the git suffix?

Reply:
For builds from git, this is expected. It is also a good indicator that this shouldn't show in any downstream packages.